src/Integration.Vsix.UnitTests/SonarLintTagger/TaggerProviderTests.cs

```diff
@@ -127,7 +128,43 @@ public void CreateTagger_should_return_null_when_analysis_is_not_supported()
         }
 
         [TestMethod]
-        public void CreateTagger_should_return_same_tagger_for_already_tracked_file()
+        public void CreateTagger_SameDocument_ShouldUseSameSingletonManager()
```

Comment:
The naming format of the other tests in this file is non-standard, so I followed our standard form for the new tests. I didn't bother changing any of the old test names.